sdk/storage/storage-blob/src/utils/utils.common.ts

```diff
@@ -461,6 +466,8 @@ export async function delay(timeInMs: number, aborter?: AbortSignalLike, abortEr
     };
 
     timeout = setTimeout(resolveHandler, timeInMs);
+    /* eslint-enable prefer-const */
```

Comment:
Use the `eslint-disable-next-line` instead of disabling and re-enabling the rule. Same for other places

Reply:
Updated.